geoapi/services/images.py

```diff
@@ -174,4 +177,6 @@ def get_exif_location(image):
         if gps_longitude_ref.values[0] != 'E':
             lon = 0 - lon
 
+    if not lat or not lon:
```

Comment:
this is the key change.  as in line 166 there is a _get_if_exist which is a different approach than before so we need to eventually check if it exists.